medusa/post_processor.py

```diff
@@ -899,39 +903,27 @@ def _is_priority(self, ep_obj, new_ep_quality):
         _, old_ep_quality = common.Quality.splitCompositeStatus(ep_obj.status)
 
         # if Medusa downloaded this on purpose we likely have a priority download
-        if self.in_history or ep_obj.status in common.Quality.SNATCHED + \
-                common.Quality.SNATCHED_PROPER + common.Quality.SNATCHED_BEST:
-            # if the episode is still in a snatched status, then we can assume we want this
-            if self.in_history:
-                self._log(u"This episode was snatched last and isn't processed yet, processing now", logger.DEBUG)
+        if self.in_history:
+
+            # If manual searched, then by pass any quality checks
+            if self.manual_searched:
+                self._log(u"This episode was manually snatched. Marking it as priority", logger.DEBUG)
                 return True
 
-            # if it's in history, we only want it if the new quality is higher or
-            # if it's a proper of equal or higher quality
-            if new_ep_quality > old_ep_quality and new_ep_quality != common.Quality.UNKNOWN:
```

Comment:
If you remove this, UNKNOWN quality will replace all other qualities as it is the highest we have (32768).